addons/account/models/account_journal_dashboard.py

```diff
@@ -333,6 +334,7 @@ def _get_open_bills_to_pay_query(self):
         return ('''
             SELECT
                 (CASE WHEN move.move_type IN ('out_refund', 'in_refund') THEN -1 ELSE 1 END) * move.amount_residual AS amount_total,
+                %(sign)s * amount_residual_signed AS amount_total_company,
```

Comment:
The `sign` thing is actually the same as the `CASE WHEN` on the above line.
But I doubt that you need it for `amount_residual_signed`

Reply:
just saw that the rest of the query  is accessing the fields by specifying the table, should I change `amount_residual_signed` to `move.amount_residual_signed` ?

Reply:
> The goal is to inverse the sign of amount_residual_signed for a purchase journal.
> For example, if you have a vendor bill, you will get a negative `amount_residual_signed`, but the line above would give a positive value (it is neither an `out_refund` or an `in_refund`).
> So in order to be coherent, you need to inverse the value to get a positive amount.
> From what I've understood.

Uh? It's exactly the same way to read the fields as in the community methods